dynamics:
  substeps: 1
  kinematic_simulation: false

solver:
  rollouts: 100
  lambda: 0.00001
  step_size: 0.01
  horizon: 1.0
  caching_factor: 0.2
  filtering: false
  cost_ratio: 0.6
  discount_factor: 1.0
  debug_print: false
  filter_type: 1
  filter_order: 3
  filter_window: 10
  verbose: false

noise_variance: [0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.001]

visualization:
  step_stride: 10
  speed_up: 4
  pause_between_rollouts: 0.01
  rollout_stride: 40
  default_visualization: false

initial_configuration: [0.0, -0.52, 0.0, -1.785, 0.0, 1.10, 0.69]
linear_weight: 10.0
angular_weight: 1.0
obstacle_radius: 0.2
